docs/src/pages/index.astro

```diff
@@ -0,0 +1,5 @@
+<script is:inline>
+  // Redirect your homepage to the first page of documentation.
+  // If you have a landing page, remove this script and add it here!
+  window.location.pathname = `/en/get-started`;
```

Comment:
I guess some version of media-chrome.org like thing would eventually replace this?